addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,97 @@
+# iCloud Binding
+
+The Apple [iCloud](https://www.apple.com/icloud/) is used to retrieve data such as the battery level or current location of one or multiple Apple devices connected to an iCloud account.
+Updates are quick and accurate without significant battery time impact.
+The Binding also offers access to the "Find My iPhone" function.
+
+An Apple account is required. Two factor authentication is supported.
+
+Please note: Application specific passwords are not supported.
+You´ll need to provide the account password to the binding.
+
+## Supported Things
+
+The following devices are known to work with this Binding:
+
+* iPhone 6s
+* iPhone 5c
+* iPhone 7
+* Apple Watch Series 2
+* iPad Air 2/2017/Pro
+* MacBook Pro
+
+Other devices connected to your iCloud account should work as well.
+Please provide feedback if you have tested another device type.
+
+## Discovery
+
+For each iCloud account to be included an iCloud Binding Thing needs to be configured with your ID and password.
```

Comment:
In the UI the thing is called "iCloud Account", so maybe revise this sentence to "An iCloud account can be added as a thing and needs to be configured with your ID and password."

Reply:
Done.